### PR TITLE
[9.0][MIG] stock_quant_manual_assign: port to 9.0

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Supersedes https://github.com/OCA/stock-logistics-warehouse/pull/173

### CODE REVIEW

File: stock_quant_manual_assign/wizard/assign_manual_quants.py [hunk 1]
```diff
@@ -17,16 +17,23 @@ def check_qty(self):
             if record.quants_lines:
                 move = self.env['stock.move'].browse(
                     self.env.context['active_id'])
-                if record.lines_qty > move.product_uom_qty:
+                move_qty = self.env['product.uom']._compute_qty_obj(
+                    move.product_uom, move.product_uom_qty,
+                    move.product_id.uom_id)
```
Comment (Garamotte) [inline stock_quant_manual_assign/wizard/assign_manual_quants.py]: You could use the `move.product_qty` field, which already does this :
https://github.com/odoo/odoo/blob/c8ca71fc62b44dbb6ba9eebf9091dae87c723cf4/addons/stock/stock.py#L1840

File: stock_quant_manual_assign/wizard/assign_manual_quants.py [hunk 2]
```diff
@@ -17,16 +17,23 @@ def check_qty(self):
...
     def _compute_qties(self):
         move = self.env['stock.move'].browse(self.env.context['active_id'])
-        lines_qty = sum(self.quants_lines.mapped('qty'))
+
+        lines_qty = sum(quant_line.qty for quant_line in self.quants_lines
+                        if quant_line.selected)
         self.lines_qty = lines_qty
-        self.move_qty = move.product_uom_qty - lines_qty
+        move_qty = self.env['product.uom']._compute_qty_obj(
+            move.product_uom, move.product_uom_qty, move.product_id.uom_id)
```
Comment (Garamotte) [inline stock_quant_manual_assign/wizard/assign_manual_quants.py]: Same comment as above, you could use the `move.product_qty` field.

**Review by Garamotte — CHANGES_REQUESTED** [review]
You should update this PR to apply this fix (from 8.0) : 33e1a3ba7e543d63f14bec2d27452a46480f3e85

**Comment by pedrobaeza** [comment]
@jbeficent, can you please attend comments and squash commits that come from the same author?

**Comment by pedrobaeza** [comment]
@jbeficent, are you going to continue with this PR?

**Comment by JordiBForgeFlow** [comment]
@pedrobaeza yes, absolutely!

**Comment by JordiBForgeFlow** [comment]
cc @eficent

**Comment by pedrobaeza** [comment]
Please attend comments from @sylvain-garancher then and we can proceed to merge this.

**Comment by JordiBForgeFlow** [comment]
@lreficent can you attend to this one please?

**Comment by LoisRForgeFlow** [comment]
Are the changes correct?

**Comment by JordiBForgeFlow** [comment]
LGTM

**Comment by Garamotte** [comment]
LGTM (code review only)

**Comment by pedrobaeza** [comment]
Can you squash all "OCA Transbot" commits all together and the commits from yourself as well?

**Comment by LoisRForgeFlow** [comment]
Our commits and OCA ones are now squashed.

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM (Tested on runbot).

**Review by jjscarafia — APPROVED** [review]
:+1: Tested on runbot

**Review by pedrobaeza — APPROVED** [review]
Merging when CIs finish after squashing a bit some commits

**Pull request merged**